### PR TITLE
Fix code styling

### PR DESCRIPTION
**Pull request by trnubo**

### CODE REVIEW

**Comment by gamesh411** [comment]
Hi, in my opinion just cut this commit in three parts (.gitignore, use env file to provide default user in the makefile, and the code style). Other than that LGTM.

**Comment by trnubo** [comment]
Good call. I've split this into two PRs, this one and #7

File: add-virtual-user.sh [hunk 1]
```diff
@@ -2,22 +2,22 @@
...
+  echo "[ -d ] Delete the database first" >&2
+  exit 1
 fi
 
-if [ "$1" == "-d" ]; then
-    if [ -f $DB ]; then
-        rm $DB
-    fi
-    shift
+if [[ "${1}" == "-d" ]]; then
```
Comment (gamesh411) [inline add-virtual-user.sh]: I think we should either use the bracketed syntax for special shell variables (i.e. $#, $0, $1 ... vs. ${#}, ${0}, ${1} ...) consistently. Use the same syntax in entry.sh as well.

**Comment by trnubo** [comment]
Going to update bash style in a new release.

**Pull request closed without merging**